src/org/usfirst/frc/team2706/robot/JoystickMap.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team2706.robot;
+
+public class JoystickMap {
```

Comment:
Looks like a cool idea! Maybe make joystick maps for different controllers, eg Xbox Controller, Logitech Controller, Normal Joystick, then do like JoystickMap.XBox.BACK_LEFT_TRIGGER or something

Reply:
Yes, that is a good plan.